.travis.yml

```diff
@@ -1,20 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.11"
+  - "8.9.3"
 
 before_install:
   - npm install -g grunt-cli
-
-notifications:
```

Comment:
Btw I can understand removing this info from our fork. At the same time, as I mentioned live, I bet Andris would take a PR. We know him personally and could reach out if you liked.

Reply:
I'm open to that, sure - however, I'd like to get this out ASAP given that it fixes a regression. Could we email him and see if we could expedite a review?

Reply:
Sure! Brad has his email.

You can also merge this into our fork now and submit a "copy" to the upstream repo, and later reconcile the two.

Reply:
Cool - emailed. I'll wait until tonight to see if he responds, otherwise i'll merge and deploy tomorrow morning.